pandas/tests/io/formats/test_format.py

```diff
@@ -2810,6 +2810,34 @@ def test_to_string_multindex_header(self):
         assert res == exp
 
 
+class TestGenericArrayFormatter:
+    def test_1d_array(self):
```

Comment:
can you add a test which more closely replicates the bug you are seing. IIRC you are essentially embedding ndim > 2 arrays in a Series / DataFrame?

Reply:
I've added an additional test case under `TestGenericArrayFormatter` in 17f167a. The new test uses a stub extension type to replicate the original problem that I described in #33770 .